scalyr_agent/util.py

```diff
@@ -78,7 +79,7 @@ def ujson_dumps_custom(obj, fp):
             :param fp: If not None, then a file-like object to which the serialized JSON will be written.
             :type obj: dict
             :return: If fp is not None, then the string representing the serialization.
-            :rtype: Python3 - six.text_type, Python2 - six.text_type | six.binary_type
+            :rtype: Python3 - six.text_type, Python2 - six.binary_type
```

Comment:
Seeing the different return types between Python3 and Python2 still throws me, though I understand, based on how this code is used, it's fine.  Let's add a comment after line 77 to something like this:

`Note, this function returns different types (text vs binary) based on which version of Python you are using.  We leave the type unchanged here because the code that invokes this function will convert it to the final desired return type.  Otherwise, we'd be double converting the result in some cases.`

Reply:
Done.